### DISCUSSION TITLE
Do .3ds files work?

### DISCUSSION THREAD

**Discussion by pizza699** [EmulatorJS/EmulatorJS#577] [General]
Hi recently got a ROM that is a .3ds file emulatorjs freezes when using the file am I doing something wrong or does emulatorjs not support it?

**Comment by ethanaobrien** [comment]
What console is this 3ds file for?

**Comment by shaylevary** [comment]
Nintendo 3DS - a Nintendo console that was a successor to the predecessor NDS.
https://docs.libretro.com/library/citra/

**Reply by ethanaobrien** [reply]
Oh, then no. The 3ds console is not supported due to a mix of power it needs and that it uses cmake

**Comment by pizza699** [comment]
okay thanks
